### ISSUE TITLE
Pull Tweets from twitter

### ISSUE THREAD

**Issue by malteserteresa** [opt-out-tools/find-out#2]

**Comment by MikeJohnPage** [comment]
Currently, there is an R script running every 15 minutes (via crontab) on an aws EC2 instance connected to an RDS MySQL database, maxing out the Twitter rate limit.

The `rtweet` package is being used to collect the data via the standard Twitter search API. I have collated a vector of search terms that get passed to the query parameter of the `search_tweets()` function like so (in the interest of not listing offensive information I have omitted the actual terms):

```r
search_terms <- c(
 "search_term_1",
 "search_term_2",
 "search_term_3"
)
```
I selected the search terms from an article in the literature (that can be found on the Wiki). However if anybody would like to propose any search terms they have stumbled across (ideally evidence based), I can add them to the list (if they are not already there), and hopefully improve the feed!
